guides/v2.1/frontend-dev-guide/themes/admin_theme_create.md

```diff
@@ -74,6 +74,27 @@ In the default `Magento/backend` theme `lib/web/images/magento-logo.svg` is used
 To override it, in your theme directory, create a `web/images` sub-directory, and add your custom file named `magento-logo.svg`. 
 If you want to use the file with other name and/or format, you need to additionally declare it as described in [Declaring theme logo]({{ page.baseurl }}/frontend-dev-guide/themes/theme-create.html#logo_declare).
 
+To update Admin theme logo, take the following steps:
+
+1. After you create admin theme then You can customize Magento admin logo.
```

Comment:
This is not a step. It is a statement. Remove it.

Reply:
@abrarpathan19, this is the third time I'm asking you to make the same revision. It is not an efficient use of time.

Reply:
The requested revision is to delete line 79.